benchmark/Makefile.am

```diff
@@ -35,9 +39,10 @@ $(LOCAL_DIR)/config.h: $(LOCAL_DIR)
 
 $(BENCHOUT): $(LOCAL_DIR)/config.h
 	cd $(LOCAL_DIR) && make clean
-	cd $(LOCAL_DIR) && $(TIMER) make
+	cd $(LOCAL_DIR) && $(TIMER) -o baseline.times make
 	cd $(LOCAL_DIR) && make clean
-	cd $(LOCAL_DIR) && $(TIMER) ../$(EXE) make
+	cd $(LOCAL_DIR) && $(TIMER) -o build-recorder.times ../$(EXE) make
+	cd $(LOCAL_DIR) && $(AWK) -f ../benchmark build-recorder.times baseline.times
```

Comment:
The last command should save the output to `$(BENCHOUT)`.
And there is no point in the running it inside `$(LOCAL_DIR)`